css/css-properties-values-api/META.yml

```diff
@@ -904,6 +921,16 @@ links:
           subtest: <transform-function> values are computed correctly [translateX(calc(11em + 10%))]
         - test: registered-property-computation.html
           subtest: <transform-function>+ values are computed correctly [translateX(10%) scale(2)]
+        - test: registered-property-initial.html
+          subtest: Initial value for <transform-function> correctly computed [rotate(42deg)]
```

Comment:
Need these

Reply:
Update: now included in https://github.com/web-platform-tests/wpt-metadata/pull/3467